src/h3lib/include/constants.h

```diff
@@ -80,5 +80,7 @@
 /** H3 index modes */
 #define H3_HEXAGON_MODE 1
 #define H3_UNIEDGE_MODE 2
+#define H3_EDGE_MODE 3
```

Comment:
the edge mode is reserved I presume?

Reply:
Yes - I added `H3_EDGE_MODE` proactively because it made more sense for it to be sequentially next to `H3_UNIEDGE_MODE`, and we're pretty sure we're going to add it. Could remove here, I just thought it would look funny to have `H3_UNIEDGE_MODE`, `H3_VERTEX_MODE`, and `H3_EDGE_MODE ` in that order.